code/lib/builder-vite/src/utils/process-preview-annotation.ts

```diff
@@ -37,5 +33,21 @@ export function processPreviewAnnotation(path: PreviewAnnotation | undefined) {
     return transformAbsPath(path);
   }
 
+  // resolve absolute paths relative to project root
+  if (isAbsolute(path)) {
+    return slash(`/${relative(projectRoot, path)}`);
+  }
```

Comment:
What about an absolute path outside the `projectRoot`? Won't that lead to different behaviour to the `../` case below?